pandas/tests/groupby/aggregate/test_aggregate.py

```diff
@@ -313,3 +313,14 @@ def test_order_aggregate_multiple_funcs():
     expected = pd.Index(['sum', 'max', 'mean', 'ohlc', 'min'])
 
     tm.assert_index_equal(result, expected)
+
+
+def test_uint64_type_handling():
+    # GH 26310
+    df1 = pd.DataFrame({'x': 6903052872240755750, 'y': [1, 2]})
+    df1.groupby('y').agg({'x': 'first'})
+    df2 = df1
+    df2.x = df2.x.astype(np.uint64)
+    df2.groupby('y').agg({'x': 'first'})
```

Comment:
we are going to want to parameterize this on an int64 as well (so another later of paramaterization)

Reply:
done